velox/external/date/tz.h

```diff
@@ -105,10 +105,10 @@ static_assert(HAS_REMOTE_API == 0 ? AUTO_DOWNLOAD == 0 : true,
 #  endif
 #endif  // HAS_DEDUCTION_GUIDES
 
-#include "date.h"
+#include "velox/external/date/date.h"
 
 #if defined(_MSC_VER) && (_MSC_VER < 1900)
-#include "tz_private.h"
+#include "velox/external/date/tz_private.h"
```

Comment:
if we're doing this, then we probably don't need to add "velox/external/date/" to the include path anymore (can clean up cmake and targets file)